blockchain/config.go

```diff
@@ -111,6 +113,7 @@ var (
 		WorkingSetCacheSize:           20,
 		StreamingBlockBufferSize:      200,
 		PersistStakingPatchBlock:      19778037,
+		FactoryDBType:                 db.DBBolt,
```

Comment:
just use `string` type would be straightforward and easier to setup in yaml file
"Bolt", "Pebble"